pubsub/gcloud/gcloud.go

```diff
@@ -76,6 +76,16 @@ func (p *Gcloud) Publish(ctx context.Context, data []byte) error {
 	return nil
 }
 
+// Publish implements the Publisher interface for publishing a message
```

Comment:
comment needs to be updated it should be `// PublishUntilComplete ...` it would also be good to mention that this is a block call

Reply:
ohhh ofc